app/admin.go

```diff
@@ -154,6 +154,18 @@ func SaveConfig(cfg *model.Config, sendConfigChangeClusterMessage bool) *model.A
 		return err
 	}
 
+	if *cfg.FileSettings.DriverName == model.IMAGE_DRIVER_S3 {
+		if len(*cfg.FileSettings.AmazonS3Region) == 0 || *cfg.FileSettings.AmazonS3Region == "" {
```

Comment:
Redundant. You should just change it to `if *cfg.FileSettings.AmazonS3Region == "" {`.

Reply:
yay, thanks!